cluster-api/src/main/java/io/aiven/klaw/clusterapi/controller/MetricsApiController.java

```diff
@@ -0,0 +1,27 @@
+package io.aiven.klaw.clusterapi.controller;
+
+import io.aiven.klaw.clusterapi.services.MetricsApiService;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/metrics")
+@Slf4j
+public class MetricsApiController {
+
+  @Autowired MetricsApiService metricsApiService;
+
+  @PostMapping(value = "/getMetrics")
+  public ResponseEntity<Map<String, String>> getMetrics(
+      @RequestBody MultiValueMap<String, String> metricsRequest) throws Exception {
+    String metricsObjectName = "kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec";
+    String jmxUrl = "service:jmx:rmi:///jndi/rmi://localhost:9996/jmxrmi";
```

Comment:
i guess this is not for test, right?
should not it be configurable?
At some env this port could be occupied

Reply:
this is not a working functionality, and still work in progress. I like to keep it, but with a config yea